Demo/Demo/Models/ProcessOrderParams.swift

```diff
@@ -0,0 +1,5 @@
+struct ProcessOrderParams: Codable {
+    let orderId: String
+    let intent: String
+    let countryCode: String
```

Comment:
question: I dont remember having this field in capture/authorize endpoints, is this required in our test server? 🤔

Reply:
Currently, the sample merchant server requires a countryCode. However, it doesn't need it. It was a requirement of a previous project.

We made a note do update & re-deploy the sample server removing it! For now, it has to stay for the request to succeeed.